app/views/shared/review_application/_income_payments_and_assets_for_bank_statement_upload.html.erb

```diff
@@ -28,34 +32,30 @@
     <% end %>
   </section>
 
-  <% if @legal_aid_application.uploading_bank_statements? %>
+  <!-- BENEFITS -->
+
+  <% if @legal_aid_application.client_uploading_bank_statements? %>
     <%= render("shared/check_answers/benefits_income",
                transactions_for: "applicant",
                individual: @legal_aid_application.applicant,
                state_benefits: @legal_aid_application.applicant.state_benefits,
                read_only: true) %>
   <% end %>
 
+  <!-- INCOME -->
+
   <section class="print-no-break">
     <%= render(
           "shared/check_answers/payments_summary",
-          payment_type: t(".credits-section-heading"),
+          payment_type: t(".credits-section-heading", individual: "your client"),
```

Comment:
Localization: should the value `your client` be contains with a locales file?

Reply:
Yes, it could be, I will update this